.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
see above.

Reply:
I didn't change this, because the official example also uses `fetch-depth`.